### PR TITLE
Lock ui

### PR DESCRIPTION
**Pull request by szerintedmi**

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit cfa345ebd777e587329aaab30dac93f3e4dd3687

https://app.netlify.com/sites/augmint/deploys/5acc89ec02ed83660cd4c231

File: src/modules/ethereum/lockTransactions.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            from: userAccount,
+            gas: gasEstimate
+        });
+
+    try {
+        const transactionHash = await processTx(tx, txName, gasEstimate);
+        return { txName, transactionHash };
+    } catch(error) {
+        throw new EthereumTransactionError(
+            "Place order failed.",
```
Comment (szerintedmi) [inline src/modules/ethereum/lockTransactions.js]: wrong error string (left from copy :) )

File: src/modules/ethereum/lockTransactions.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        return parsed;
+
+    }, []);
+
+    return products;
+}
+
+export async function newLockTx(productId, lockAmount) {
+    const txName = "New lock";
+    const getState = store.getState;
```
Comment (szerintedmi) [inline src/modules/ethereum/lockTransactions.js]: is it needed? why not store.getState().xxx directly? (nitpicking)

**Comment by szerintedmi** [comment]
closing this and reopening b/c of some merge weirdness:
  command line merge says it's up to date but GitHub ui still says it's conflicting

**Comment by szerintedmi** [comment]
opening a new PR didn't help. so reopening this to keep review history

**Comment by reyrodrigez** [comment]
I'll do a rebase to clean up my commit messages

**Comment by szerintedmi** [comment]
@reyrodrigez : hold on, I screwed up the merge, just trying to revert

**Comment by szerintedmi** [comment]
This is PR is abandoned b/c of merge mess up (lock-ui-mergefix branch is used)

**Pull request closed without merging**